examples/01-advanced-postprocessing/Maxwell2D_Transient.py

```diff
@@ -2,13 +2,15 @@
 
 Maxwell 2D Analysis
 -------------------
-This example shows how you can use PyAedt to create a project in
-in Maxwell 2D and run a transient simulation.
+This example shows how you can use PyAedt to create a project in Maxwell 2D
+and run a transient simulation.
 
-To provide the advanced postprocessing features needed for this example, Matplotlib, NumPy, and 
-PyVista must be installed on the machine.
+To provide the advanced postprocessing features needed for this example,
+Matplotlib: <https://pypi.org/project/matplotlib/>,
+NumPy: <https://pypi.org/project/numpy/>,
+and PyVista https://pypi.org/project/pyvista/ must be installed on the machine.
```

Comment:
Please use rst style documentation for the links:

```suggestion
The following libraries are required for the advanced post-processing features
used in this example:

- `matplotlib <https://pypi.org/project/matplotlib/>`_
- `NumPy <https://pypi.org/project/numpy/>`_
- `PyVista https://pypi.org/project/pyvista/`_

Install these with:

.. code::

   pip install numpy pyvista matplotlib

```

Reply:
@akaszynski I did that the first time and then switched back to the current version. Thanks for the feedback. I will do the modifications.